static/_redirects

```diff
@@ -1,6 +1,6 @@
 # See https://www.netlify.com/docs/redirects/
 
-# Move guides into their own paths
+# Older (prev 5.0 doc URLs)
```

Comment:
Should we update the old redirects as well to avoid multiple redirects? Or is this legacy at this point?

Reply:
Doesn't seem worth the bother. I'd consider deleting them.